### PR TITLE
Rename it.json to it_it.json

### PR DESCRIPTION
**Pull request by gerardofallani**
The it.json file was totally in English, so instead of updating it I just made a new one. I hope it's ok. Please, let me know for the future.

### CODE REVIEW

**Comment by fnoks** [comment]
It needs to be named it.json. If not, it will never be used. Please rename it to it.json. Also, it is not legal JSON.

**Comment by gerardofallani** [comment]
> It needs to be named it.json. If not, it will never be used. Please rename it to it.json. Also, it is not legal JSON.

I had done if I could, but it seems I can't, since the it.json file already exists.
Schermata 2019-04-24 alle 11 24 32
And of course I can't delete it.
So the new it-it.json contains the Italian translation. The original it.json is entirely written in English.
What I could do is to update the original file with the new (Italian) content.

**Comment by gerardofallani** [comment]
I hope I've made some order.
I moved the changes I did on it_it.json to it.json file, updating it, and then I deleted it_it.json.

File: language/it.json [hunk 1]
```diff
@@ -27,40 +27,40 @@
...
+              "label": "Mostra il feedback come popup"
             },
             {
-              "label": "Localization",
+              "label": "Localizzazione",
               "fields": [
                 {
-                  "label": "Retry button text",
-                  "default": "Retry"
+                  "label": "Testo del pulsante \"Riprova\",
```
Comment (fnoks) [inline language/it.json]: You are missing a " here.

Comment (gerardofallani) [reply language/it.json]: I fixed it.

**Pull request merged**